pkg/dogstatsd/server.go

```diff
@@ -249,16 +249,12 @@ func (s *Server) parsePacket(packet *listeners.Packet, metricSamples []*metrics.
 	extraTags := s.extraTags
 
 	if packet.Origin != listeners.NoOrigin {
-		log.Tracef("Dogstatsd receive from %s: %s", packet.Origin, packet.Contents)
```

Comment:
Why remove the Tracef here ? Not useful ?

Reply:
Was useful in when the feature was being develop, but it's too verbose, when you have significant DSD traffic the logger can't keep up and the memory of the agent sky rockets.